llvm_jni/main.bal

```diff
@@ -0,0 +1,13 @@
+import llvm_jni.llvm;
```

Comment:
Our current llvm module is `wso2/nballerina.nback.llvm`.  So I am thinking we should rename it to something like
`wso2/nballerina.print.llvm` (i.e. it interfaces to LLVM by printing) and this one should be `wso2/nballerina.jni.llvm` (i.e. interfaces to LLVM by JNI).

Reply:
Fixed